### ISSUE TITLE
Add ORCID IDs to author list

### ISSUE THREAD

**Issue by ehsteve** [heliophysicsPy/standards#4]
It would be good to add ORCID id to the author list.

**Comment by ehsteve** [comment]
Note that ORCID id do show up on the [Zenodo page](https://zenodo.org/record/2529131#.XCuQvS2ZN26). Not sure if others can add their own ids. Please let me know.

**Comment by BaptisteCecconi** [comment]
I agree that adding the ORCID would be a very good idea. I checked the Zenodo page, but only yours shows up. How could I add mine ?

**Comment by jibarnum** [comment]
@sapols this is a long-since stale issue, BUT, I think it has applicability still in PyHC in general. Should we add somewhere to the Contacts page an ORCID addition?

**Comment by sapols** [comment]
@jibarnum Just now seeing this. I can imagine including ORCIDs on our [People page](https://heliopython.org/people/). The implementation would be pretty simple, but I can't think of a way to automate people adding their own. Best idea I have is I implement the feature then ask everyone to throw their ORCIDs in this [members.yml](https://github.com/heliophysicsPy/heliophysicsPy.github.io/blob/main/_data/members.yml) file.

Can you clarify your idea about something on the Contacts page?

**Comment by jibarnum** [comment]
> @jibarnum Just now seeing this. I can imagine including ORCIDs on our [People page](https://heliopython.org/people/). The implementation would be pretty simple, but I can't think of a way to automate people adding their own. Best idea I have is I implement the feature then ask everyone to throw their ORCIDs in this [members.yml](https://github.com/heliophysicsPy/heliophysicsPy.github.io/blob/main/_data/members.yml) file.
> 
> Can you clarify your idea about something on the Contacts page?

Sorry about that. I typed Contacts, but meant People page.

**Comment by sapols** [comment]
Cool, I'll add this to my TODO list. It'll be pretty far down the list, but it'll be on there...
